wrappers/Julia/CoolProp.jl

```diff
@@ -239,6 +282,9 @@ function AbstractState_keyed_output(handle::Clong, param::Clong)
   end
   return output
 end
+function AbstractState_output(handle::Clong, param::AbstractString)
+  return AbstractState_keyed_output(handle, get_param_index(param))
+end
```

Comment:
Overloading the function to be able to give the output name in the form of a string (calling internally get_param_index).
This allows to write code faster when only one output with this name is required.